### PR TITLE
Renamed fields in FlaskSqlAlchemyTestConfiguration in test_filters.py.

### PR DESCRIPTION
**Pull request by rosealti**

### CODE REVIEW

File: python/tests/api/routes/test_filters.py [hunk 1]
```diff
@@ -41,45 +41,49 @@ def create_app(self):
...
-                                             ssh_prnamt_type='None',
-                                             put_call='None',
-                                             investment_discretion='SOLE',
-                                             other_manager='None',
-                                             voting_authority_sole=22967078,
-                                             voting_authority_shared=0,
-                                             voting_authority_none=0
-                                             )]
-        self.edgar_filing1_row2 = EdgarFiling(accession_no=ACCESSION_NO_TABLE1_ROW2, cik_no=COMPANY_CIK_1,
+        self.edgar_filing1_row2 = EdgarFiling(accession_no=ACCESSION_NO_TABLE1_ROW2,
```
Comment (datdo) [inline python/tests/api/routes/test_filters.py:49]: personally, I like ACCESSION_NO_2 better

File: python/tests/api/routes/test_filters.py [hunk 2]
```diff
@@ -41,45 +41,49 @@ def create_app(self):
...
-                                             put_call='None',
-                                             investment_discretion='SOLE',
-                                             other_manager='None',
-                                             voting_authority_sole=4257078,
-                                             voting_authority_shared=0,
-                                             voting_authority_none=0
-                                             )]
-
-        self.company2 = Company(cik_no=COMPANY_CIK_2, company_name="Nice Industries", filing_count=1)
+        self.data_13f1_row1 = [Data13f(equity_holdings_id="67896567",
```
Comment (datdo) [inline python/tests/api/routes/test_filters.py:52]: I don't understand why this change was made. Is this what you mean?:

```python
        self.data_13f_table1_row1 = Data13f(...
        self.data_13f_table1_row2 = Data13f(...
        self.data_13f_table1 = [self.data_13f_table1_row1, self.data_13f_table1_row2 ]
       ...
       send_data_to_db(self.company1, self.edgar_filing1_row1, self.data_13f_table1)
```

**Review by datdo — CHANGES_REQUESTED** [review]

File: python/tests/api/routes/test_filters.py (hunk 1 above)
Comment (rosealti) [reply python/tests/api/routes/test_filters.py]: Done

File: python/tests/api/routes/test_filters.py [hunk 3]
```diff
@@ -41,88 +42,95 @@ def create_app(self):
...
+                                              ssh_prnamt_type='None',
+                                              put_call='None',
+                                              investment_discretion='SOLE',
+                                              other_manager='None',
+                                              voting_authority_sole=9823278,
+                                              voting_authority_shared=0,
+                                              voting_authority_none=0
+                                              )]
+
+        send_data_to_db(self.company_cool, self.edgar_filing_cool, self.data_13f_cool)
```
Comment (datdo) [inline python/tests/api/routes/test_filters.py:130]: don't use send_data_to_db

**Review by datdo — CHANGES_REQUESTED** [review]

File: python/tests/api/routes/test_filters.py [hunk 4]
```diff
@@ -41,88 +42,109 @@ def create_app(self):
...
+                                              ssh_prnamt_type='None',
+                                              put_call='None',
+                                              investment_discretion='SOLE',
+                                              other_manager='None',
+                                              voting_authority_sole=9823278,
+                                              voting_authority_shared=0,
+                                              voting_authority_none=0
+                                              )]
+
+        db.session.merge(self.company_cool)
```
Comment (datdo) [inline python/tests/api/routes/test_filters.py:130]: why merge instead of add?

**Review by datdo — CHANGES_REQUESTED** [review]

**Review by datdo — APPROVED** [review]

**Pull request merged**